### ISSUE TITLE
ast-tool-py install procedure for offline server

### ISSUE THREAD

**Issue by zoranbosnjak** [zoranbosnjak/asterix-tool#2]
Install procedure is required for the case if a target server is offline (without internet connection). That is:
- download/prepare packages including dependencies on the server with internet connection
- transfer environment to offline server (copy file/dir)
- install on offline server

Maybe related to this problem... there seems to be a problem with `python3 -m build`  on ubuntu-22.04, as discussed here:

- https://github.com/pypa/build/issues/215
- https://github.com/deadsnakes/issues/issues/198
- https://bugs.launchpad.net/ubuntu/+source/python-build/+bug/1992108

... and some workaround to upgrade 'build':

```bash
python3 -m pip install --upgrade build
```

**Comment by zoranbosnjak** [comment]
It looks like `pip` version is the problem, as discussed here:
<https://stackoverflow.com/questions/25265334/python-packages-are-getting-installed-as-unknown-using-pip>

The solution is to upgrade `pip` on both, the auxilary server (with internet access) and the tar get server. At the time of wirting the current (working) `pip` version is `23.3.1`.

```bash
python3 -m pip install --upgrade pip
```

**Comment by zoranbosnjak** [comment]
Resolved with commit 5212f92.
